mlrun/serving/routers.py

```diff
@@ -375,9 +400,10 @@ def __init__(
         url_prefix: str = None,
         health_prefix: str = None,
         vote_type: str = None,
-        weights: Union[List[float], Dict[str, float]] = None,
+        weights: Dict[str, float] = None,
```

Comment:
Shouldn't we accept a list of floats as well? it will be assigned with the order of the models that were added using `add_model`. What do you think?

Reply:
We talked about it and maybe will add it in the next one

Reply:
would keep as dict, so the results are as expected, less user errors ..